spmd/compiler/distribute.py

```diff
@@ -165,22 +165,17 @@ def remap_arg(arg: object) -> object:
     def unwrap_local(e: object) -> object:
         return e._local_tensor if isinstance(e, DTensor) else e
 
-    gm = make_fx(dispatch)(local_target_args)
-    for nd in gm.graph.nodes:
-        if nd.name == "_tensor_constant0":
-            print(
-                f"WARNING: found constant while SPMD expansion for {node}. Graph {gm}. This is not supported."
-            )
-    return gm
```

Comment:
I realized this warning is wrong, i think a constnat is always expected (It's the ProcessGroup i think).